_posts/2018-02-01-vue.js-import-paths.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Vue Import Paths"
+date: 2018-02-01 13:05:34
+tags:
+- Web
+- Vue
+- JavaScript
+- Webpack
+categories:
+twitter_text: "Vue root path with webpack"
+authors: Andrew Rady
+---
+
+In today’s front-end ecosystem we are breaking down code into separate files and importing them. This helps us separate out code and reuse it more effectively. Vue does this in components similar to the way React does it. We have been using Vue in our projects lately and we love it so far because it’s progressive in nature. We can plug it into existing applications and add new features while giving the user a smoother experience, to creating full blown SPAs with it. We have run into a few snags, not completely the fault of Vue. Importing components from obscure locations is something that has been around for some time. Dealing with relative paths is not a problem for smaller projects but can become an issue as the project grows. With a little tweaking to webpack file we can implement a “root path” into our Vue applications. 
```

Comment:
A couple things I might reword here:
`We can plug it into existing applications and add new features while giving the user a smoother experience, to creating full blown SPAs with it.`
something more like, so we get the from -> to in the sentence
`We can do a variety of things from plugging it into existing applications to add new features and give the user a smoother experience to creating full blown SPAs with it.`

`With a little tweaking to webpack file we can implement a “root path” into our Vue applications.`
`With a little tweaking to our webpack file we can implement a "root path" into our Vue applications.`